### PR TITLE
Build and export messges.pot to a separate package

### PR DESCRIPTION
**Pull request by Ealdwulf**
@tombettany

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -11,3 +11,8 @@ Depends: ${misc:Depends}, x11vnc, libkdesk-dev, kano-settings (>=2.0-3), gettext
 Suggests: kano-profile
 Description: Simple VNC server wrapper
  Allows really simple toggling of the tightvncserver.
+
+Package: kano-vnc-i18n-orig
+Architecture: all
+Description: Data for working on translations of kano-vnc
+Multi-Arch: foreign
```
Comment (tombettany) [inline debian/control]: Just pointing it out

File: debian/kano-vnc-i18n-orig.install [hunk 2]
```diff
@@ -0,0 +1 @@
+po/messages.pot /mnt/translations/translations/kano-vnc/
```
Comment (skarbat) [inline debian/kano-vnc-i18n-orig.install]: My only concern here is we are populating a hardcoded mount point.
How are we supposed to explain what to do to anyone who wants to clone and `debuild` on their own.

**Comment by Ealdwulf** [comment]
I don't think this casuses a problem to someone who wants to debuild - only
if they want to install the package. In princpal we could install them
someonwere in `/usr` and make a symlink there on the server to
`/mnt/translations`. But I've already merged a bunch of these.

**Comment by skarbat** [comment]
Oh I see, no harm then if it `debuilds` fine then, @Ealdwulf  👍

**Pull request merged**